### PR TITLE
[terra-core] Removed inline style for dev-site components Part 1

### LINKED ISSUE #2386: Remove inline styles in dev site components
# Feature Request

## Description

We want to remove inline styles from our dev site components to set a better example for consumers of terra.  As a replacement, we should pass in classes and use css to style the components.

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
Removed all inline styles from dev-site and replaced the same styles with external CSS for the following packages.

1. Action Footer
2. ActionHeader
3. Arrange
4. Avatar
5. Button Group
6. Card

Partially resolves #2386

### CODE REVIEW

File: packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    display: block;
+    height: 20px;
+    width: 40px;
+  }
+
+  .text-with-blue-border {
+    border: 2px solid #0079be;
+  }
+
+  .align-example-div {
```
Comment (bjankord) [inline packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: nit: Let's change this to `placeholder`

File: packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @include align-example-common();
+  }
+
+  .align-example-div-blue {
```
Comment (bjankord) [inline packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: nit: Let's change this to `highlighted-placeholder`

File: packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+:local {
+  @mixin align-example-common() {
+    background-color: #000;
+    display: block;
+    height: 20px;
+    width: 40px;
+  }
+
+  .text-with-blue-border {
```
Comment (bjankord) [inline packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: nit: Let's change this to `outlined-placeholder`

Comment (pranav300) [reply packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: Fixed here https://github.com/cerner/terra-core/pull/2427/commits/1b2f4166cbfec7f0bcf85035c64e83ec7746a85f.

File: packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss (hunk 2 above)
Comment (pranav300) [reply packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: Fixed here https://github.com/cerner/terra-core/pull/2427/commits/1b2f4166cbfec7f0bcf85035c64e83ec7746a85f.

File: packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss (hunk 1 above)
Comment (pranav300) [reply packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss]: Fixed here https://github.com/cerner/terra-core/pull/2427/commits/1b2f4166cbfec7f0bcf85035c64e83ec7746a85f.

**Comment by pranav300** [comment]
> The [Arrange jest test uses inline styles](https://github.com/cerner/terra-core/blob/25e2a2787a659556df40a695d999d1881439f583/packages/terra-arrange/tests/jest/Arrange.test.jsx#L47), but it is just a jest test so I'm not quite sure if that is what we are concerned with.

We are only removing inline styles from the dev-site components.

**Pull request merged**